plugin/storage/s3/s3.go

```diff
@@ -55,7 +56,7 @@ func (client *Client) UploadFile(ctx context.Context, filename string, fileType
 	uploader := manager.NewUploader(client.Client)
 	uploadOutput, err := uploader.Upload(ctx, &awss3.PutObjectInput{
 		Bucket:      aws.String(client.Config.Bucket),
-		Key:         aws.String(filename),
+		Key:         aws.String(client.Config.Path + filename),
```

Comment:
What about using `path.Join` instead of the string concatenation? (e.g., If I set the path variable to `path` but not `path/`)

Reply:
that's cool, but what if you want to set your key with `path/subpath`?